docs/Translation.md

```diff
@@ -130,6 +130,60 @@ const App = () => (
 
 Check [the translation setup documentation](./TranslationSetup.md) for details about `ra-i18n-polyglot` and how to configure it.
 
+## `ra-i18n-i18next`
+
+React-admin also provides a package called `ra-i18n-i18next` that leverages [the i18next library](https://www.i18next.com/) to build an `i18nProvider` based on a dictionary of translations.
+
```

Comment:
Perhaps you could say a few words about why one would prefer this package over the polyglot one?

Reply:
Added